delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/SchemasRoutes.scala

```diff
@@ -124,6 +124,16 @@ final class SchemasRoutes(
                       )
                     }
                   },
+                  (pathPrefix("refresh") & put & pathEndOrSingleSlash) {
+                    operationName(s"$prefixSegment/schemas/{org}/{project}/{id}/refresh") {
+                      authorizeFor(ref, Write).apply {
+                        emit(
+                          OK,
```

Comment:
Is there a reason to make the ok explicit ?

Reply:
No, but it's perhaps more consistent with other endpoints